cob_script_server/msg/ScriptState.msg

```diff
@@ -1,5 +1,5 @@
 Header header
-int16 number
+uint32 number
```

Comment:
did we not agree to "roundtrip" the increment?

Reply:
We kind of did indeed :-)

However, that requires, on the Python side in SSS, to know what the maximum value is so you can actually round-trip when the value is too large. 
AFAIK, there is no way to get the max field size on a ROS msg and I don't like hardcoding a maximum value, because then we'll still have an issue when the message definition changes somehow. The exception is aonly raised when serializing and not when assigning (because it's just a Python `int`).

That logic (although very simple) will need testing and has a small risk of some sort of race condition when receiving multiple requests in parallel (unless we use a lock on the global variable, adding a tiny bit more complexity)

If you still prefer round-tripping, can be done of course.